src/api/visualizer.coffee

```diff
@@ -4,12 +4,12 @@ exports.getLatestEvents = ->
   try
     rtDate = new Date(Number(receivedTime))
     result = yield VisualizerEvent.find({ 'created': { '$gte': rtDate } }).sort({ 'ts': 1 }).exec()
-    this.body = { events: result }
+    @body = { events: result }
   catch err
     logAndSetResponse this, 'internal server error', "Could not fetch the latest visualizer events via the API: #{err}", 'error'
 
 exports.sync = ->
   try
-    this.body = { now: Date.now() }
+    @body = { now: Date.now() }
```

Comment:
The braces aren't really necessary anymore: `this.body = now: Date.now()` should work fine. But not a big deal